.eslintrc.json

```diff
@@ -42,6 +42,7 @@
     "jsx-a11y/no-noninteractive-element-to-interactive-role": 0,
     "class-methods-use-this": 0,
     "jsx-a11y/interactive-supports-focus": 0,
-    "jsx-a11y/click-events-have-key-events": 0
+    "jsx-a11y/click-events-have-key-events": 0,
+    "consistent-return": 0
```

Comment:
This has always caused issues, mainly in react components where you might not want to return anything:
ex: 
```
someMethod() {
  if (!props.value) return;
  
  doStuff()
}
```